Xamarin.Forms.Platform.Android/Cells/TextCellRenderer.cs

```diff
@@ -22,6 +22,8 @@ protected override AView GetCellCore(Cell item, AView convertView, ViewGroup par
 			UpdateIsEnabled();
 			UpdateFlowDirection();
 			UpdateAutomationId();
+			UpdateBackgroundColor();
+			UpdateBackgroundColor(View, Cell);
```

Comment:
I think there should be only one `UpdateBackgroundColor`

Reply:
Yep! Fixed, thanks